### PR TITLE
Added some missing change log

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

**Comment by Nyholm** [comment]
Excellent and dev test passes with sf4

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,14 +6,18 @@ The change log describes what is "Added", "Removed", "Changed" or "Fixed" betwee
...
-- `Http\Client\Common\PluginClientFactory` factory service.
+- Add missing service reference for `cache_key_generator`
+- Symfony 4 support. 
+- `Http\HttplugBundle\Collector\Collector::reset()`
 
 ### Changed
 
 - `ProfilePlugin` and `StackPlugin` are no longer registered as (private) services decorators. Those decorators are now
 created through the `Http\HttplugBundle\Collector\PluginClientFactory`.
+- `PluginClient` is not used for discovered clients.
```
Comment (dbu) [inline CHANGELOG.md]: this is more confusing than helpful, i think. we should either consider this an internal detail or explain a bit what it means.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,14 +6,18 @@ The change log describes what is "Added", "Removed", "Changed" or "Fixed" betwee
 
 ### Added
 
+- Service for`Http\Client\Common\PluginClientFactory`.
 - Any third party library using `Http\Client\Common\PluginClientFactory` to create `Http\Client\Common\PluginClient`
 instances now gets zero config profiling.
-- `Http\Client\Common\PluginClientFactory` factory service.
+- Add missing service reference for `cache_key_generator`
```
Comment (dbu) [inline CHANGELOG.md]: i think this is a bugfix. its to make a configuration option that existed but was not working actually work as expected. ok if i move it and make it more explicit?

Comment (dbu) [reply CHANGELOG.md]: oh, its already in the list of bugfixes for this version. i will remove this line.

**Comment by dbu** [comment]
see #233 where i try to implement my feedback

**Pull request merged**